frontend/app/src/locales/en.json

```diff
@@ -893,6 +893,7 @@
       },
       "generate": {
         "error": {
+          "description": "An unexpected response was received while generating the profit and loss report. This might indicate an issue with the generation mechanism. Please check the logs for any error messages (you might have to enable debug logging first) and create an issue. {error}",
```

Comment:
Can you add a newline between this text and the error?

Reply:
I can but it is a bit more complicated. I cannot directly add a newline or `<br/>` to the text resource. They will render as plain text for obvious reasons. I need to change both the dialog and the state to keep the text and error as separate content and render them via the template.